src/hearings/converters/hearing-response-length.answer.converter.ts

```diff
@@ -9,10 +9,10 @@ export class HearingResponseLengthAnswerConverter implements AnswerConverter {
   public transformAnswer(hearingState$: Observable<State>): Observable<string> {
     return hearingState$.pipe(
       map(state => {
-        const startTime = moment(state.hearingRequest.hearingRequestMainModel.hearingResponse.hearingDaySchedule.hearingStartDateTime);
-        const endTime = moment(state.hearingRequest.hearingRequestMainModel.hearingResponse.hearingDaySchedule.hearingEndDateTime);
+        const startTime = moment(state.hearingRequest.hearingRequestMainModel.hearingResponse.hearingDaySchedule[0].hearingStartDateTime);
```

Comment:
do we know the hearingdayschedule will be there all the time and if not we may have check hearingDaySchedule.length before pick up the 1st array

Reply:
Fixed